cmd/captain/run.go

```diff
@@ -324,7 +324,7 @@ func bindRunCmdFlags(cfg Config, cliArgs CliArgs) Config {
 			suiteConfig.Output.Quiet = true
 		}
 
-		if cliArgs.reporters != nil {
+		if len(cliArgs.reporters) > 0 {
```

Comment:
I don't totally understand the implications here: is this fixing a bug? if so can you add a regression test

Reply:
It's fixing a bug, but I can't add a regression test until we discuss https://rwxhq.slack.com/archives/C04J2EA6WCB/p1680816911816679 and https://rwxhq.slack.com/archives/C04J2EA6WCB/p1680817661697899 further.

I have a WIP PR here, but that even doesn't completely cover the problem because we don't have any tests covering the config file: https://github.com/rwx-research/captain/pull/17

TL;DR: reporters from the config file were never respected because `cliArgs.reporters` is _always_ not `nil` (it's an empty array instead). This change:

1. Accounted for that by checking `len` instead
2. Changed the behavior to be additive (merging the CLI reporters with the ones in the config file